challenge_restaurant/src/challenge_restaurant/restaurant.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, robot):
                                    transitions={'done': 'WAIT_FOR_CUSTOMER'})
 
             smach.StateMachine.add('WAIT_FOR_CUSTOMER',
-                                   WaitForCustomer(robot, caller_id),
```

Comment:
Can we instantly switch between these? It would be nice to be able to do both with no code changes.